.github/workflows/project_bot.yaml

```diff
@@ -0,0 +1,50 @@
+name: Add a new item to project
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+jobs:
+  add_item:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Issue an access token
+        uses: cybozu/octoken-action@v1
+        id: create-iat
+        with:
+          github_app_id: ${{ secrets.PROJECT_APP_ID }}
+          github_app_private_key: ${{ secrets.PROJECT_APP_PEM }}
+      - name: Get project data
+        env:
+          GITHUB_TOKEN: ${{ steps.create-iat.outputs.token }}
+          ORGANIZATION: topolvm
+          PROJECT_NUMBER: "2"
+        run: |
+          proj_id="$(gh api graphql -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectNext(number: $number) {
```

Comment:
`projectNext` looks deprecated. Is that OK?
https://docs.github.com/en/graphql/reference/objects#projectnext